rpc/common/moc_common_common.pb.go

```diff
@@ -213,6 +213,37 @@ func (HealthState) EnumDescriptor() ([]byte, []int) {
 	return fileDescriptor_681f78e46755eb93, []int{3}
 }
 
+type Priority int32
+
+const (
+	Priority_DEFAULT Priority = 0
+	Priority_LOW     Priority = 1
+	Priority_MEDIUM  Priority = 2
+	Priority_HIGH    Priority = 3
+)
+
+var Priority_name = map[int32]string{
```

Comment:
These maps are auto generated and can be used in layers below to covert from int to string and viceversa

Reply:
For converting string to int type, refer to example below for simple conversion from LOW to Priority_LOW type
https://github.com/microsoft/moccli/blob/72cfda4a12623d19e34961cfc2abb2efa528f63b/cmd/mocctl/compute/placementgroup/create/placementgroup_create.go#L90